persistence/src/main/java/org/mskcc/smile/persistence/jpa/CrdbRepository.java

```diff
@@ -16,4 +17,8 @@ public interface CrdbRepository extends CrudRepository<CrdbMappingModel, Long> {
             nativeQuery = true)
     Object getCmoPatientIdByInputId(@Param("inputId") String inputId);
     // another query option
+    @Query(value = "SELECT CMO_ID, DMP_ID, PT_MRN "
+           + "FROM CRDB_CMO_DMP_MAP WHERE :inputId IN (DMP_ID, PT_MRN, CMO_ID)",
+           nativeQuery = true)
+    Object getCrdbCrosswalkTripletByInputId(@Param("inputId") String inputId);
```

Comment:
Am I understanding it correctly that the CMO-DMP-MRN mapping already exists inside CRDB, and that you just wrote a new query in order to get the triplets?

Reply:
Yes, there is a crosswalk table in CRDB with this triplet info.